robottests/translation.txt

```diff
@@ -29,6 +29,7 @@ Can switch to other language  [Arguments]  ${button}  ${language}
 Enter Translation
     Open Browser  ${BASE_URL}  ${BROWSER}
     Click Link  Translations
+    Set Browser Implicit Wait     0.2 seconds
```

Comment:
What is an implicit wait? Is it possible to change this to a polling and not a sleeping?